alb.tf

```diff
@@ -0,0 +1,64 @@
+resource "aws_lb" "test" {
```

Comment:
A good advise is to choose some random, personal word like "anaranjado", and use it instead of "test". Mades the debugging process easier (think when you have a testing framework and try grepping the word "test"...)